codelab-final-state/functions/test_all.js

```diff
@@ -1,241 +0,0 @@
-// Copyright 2019 Google LLC
```

Comment:
I deleted this file since it's not part of the codelab and we don't maintain or test it.